### ISSUE TITLE
Save password auto dismiss in iOS

### ISSUE THREAD

**Issue by sparks2209** [AzureAD/microsoft-authentication-library-for-objc#2084]
Hi, can someone help what will be potential fix for this issue, as today by using webViewParameters.webviewType = .safariViewController, the popup for Save Password disappears automatically. And this popup comes only once.

**Comment by kaisong1990** [comment]
This is a duplicate of the following. Closing.

https://github.com/AzureAD/microsoft-authentication-library-for-objc/issues/848

**Comment by kaisong1990** [comment]
@sparks2209 please feel free to post questions and follow up from the above-mentioned thread. Thanks

**Comment by kaisong1990** [comment]
@sparks2209 please feel free to post questions and follow up from the above mentioned thread. Thanks
